README.md

```diff
@@ -1,288 +1,5 @@
-<a name="readme-top"></a>
+# FCC Test Suite Template
```

Comment:
I think you should use a README file that describes this project's functionalities and how to use it. This way it will be easier for people understand and contribute. You can use this README file template here: https://github.com/microverseinc/readme-template.